engine/apps/grafana_plugin/helpers/client.py

```diff
@@ -228,21 +238,31 @@ def get_users_permissions(self) -> typing.Optional[UserPermissionsDict]:
                 ]
             }
         }
+
+        We make two Grafana API calls (see `_get_user_permissions`). One for permissions prefixed with
+        `grafana-oncall-app` and another for permissions prefixed with `grafana-irm-app`.
+        We then merge these permissions together.
         """
-        response, _ = self.api_get(self.USER_PERMISSION_ENDPOINT)
-        if response is None or isinstance(response, list):
+        oncall_permissions = self._get_user_permissions(self.USER_PERMISSION_ENDPOINT_ONCALL)
+        irm_permissions = self._get_user_permissions(self.USER_PERMISSION_ENDPOINT_IRM)
```

Comment:
double checked what happens when you call this for a plugin that doesn't yet exist:
```python3
>>> from apps.user_management.models import Organization
>>> from apps.grafana_plugin.helpers.client import GrafanaAPIClient
>>> org = Organization.objects.get(stack_slug="my_stack")
>>> client = GrafanaAPIClient(api_url=org.grafana_url, api_token=org.api_token)
>>> client.api_get("api/access-control/users/permissions/search?actionPrefix=grafana-irm-app")
({}, {'url': 'https://<my_stack>/api/access-control/users/permissions/search?actionPrefix=grafana-irm-app', 'connected': True, 'status_code': 200, 'message': 'OK'})
```